source/community/policy/editorial.md

```diff
@@ -59,66 +61,75 @@ Specification versions prior to 1.0 should be considered experimental, are not s
 
 The key words _MUST_, _MUST NOT_, _REQUIRED_, _SHALL_, _SHALL NOT_, _SHOULD_, _SHOULD NOT_, _RECOMMENDED_, and _OPTIONAL_ in this document are to be interpreted as described in [RFC 2119][rfc-2119].
```

Comment:
Let's do a tidy here to the subset of words we use: _MUST_, _MUST NOT_, _SHOULD_, _SHOULD NOT_ and _MAY_ seem sufficient (we don't actually use _SHOULD NOT_ at the moment but it is in our standard subset)

Reply:
I just pushed a change that should address this

Reply:
Looks good, resolving